src/utils/stories.ts

```diff
@@ -6,8 +6,8 @@ import iconSelector from 'components/Icon/assets/iconSelector';
 export const getIconSelectorKnob = (propName: string, initialValue?: AcceptedIconNames) =>
   select(
     propName,
-    Object.keys(iconSelector)
+    Object.keys({ ...iconSelector, '': '' })
```

Comment:
can we give a `no icon` name to that because it took me awhile to understand it

Reply:
select know accepts an array of strings, so if we add `no icon` it will throw an error trying to find an icon named `no icon`. This is why empty string is added as an option (if empty string is passed as icon prop, no icon is rendered)

Reply:
please leave a comment on the code then